@codex please fix

```
building client (vite) 
 ✓ tests/navigateWithRetry.test.ts (2 tests) 11ms
 ✓ tests/k8sEnvironmentOverlays.test.ts (2 tests) 4ms
 ✓ tests/noConflictMarkers.test.ts (1 test) 1176ms
   ✓ repository sanity ‑ no merge-conflict markers > contains no <<<<<<< or >>>>>>> markers  1173ms
 ✓ scripts/tests/questDependencies.test.ts (2 tests) 5ms
 ✓ tests/svelteResolution.test.ts (4 tests) 447ms
 ✓ tests/architectureNotesAccessibilityBaseline.test.ts (2 tests) 8ms
 ✓ tests/questDependencyReferences.test.ts (1 test) 42ms
 ✓ tests/questIdPathConsistency.test.ts (1 test) 36ms
 ✓ tests/svelteSubpathResolver.test.ts (2 tests) 5ms
 ✓ tests/serviceWorkerConfigCache.test.ts (1 test) 6ms
 ✓ tests/noDuplicateItemIds.test.ts (1 test) 4ms
 ✓ frontend/tests/docsMarkdownSafety.test.ts (1 test) 9ms
 ✓ tests/questReliabilityCoverage.test.ts (1 test) 221ms
 ✓ tests/v3ReleaseStateDoc.test.ts (3 tests) 5ms
 ✓ frontend/tests/questListClassifier.test.ts (3 tests) 8ms
 ✓ tests/changelogQuestBranchingNote.test.ts (1 test) 6ms
 ✓ tests/metricsEndpoint.test.ts (3 tests) 16ms
 ✓ tests/docsSearchFeatures.test.ts (2 tests) 6ms
 ✓ tests/questProcessRecoveryPaths.test.ts (1 test) 192ms
 ✓ tests/questsPolishPromptSections.test.ts (3 tests) 6ms
 ✓ tests/builtinQuestSchema.test.ts (1 test) 103ms
 ✓ tests/githubTokenPersistence.test.ts (1 test) 14ms
 ✓ tests/spellcheckConfig.test.ts (1 test) 5ms
 ✓ frontend/src/components/__tests__/QuestLinkChips.spec.ts (2 tests) 182ms
 ✓ tests/pnpmFrontendFilter.test.ts (1 test) 17ms
03:46:04 [build] Rearranging server assets...
03:46:04 [build] Server built in 34.95s
03:46:04 [build] Complete!
 ✓ tests/buildOutputValidation.test.ts (1 test) 41751ms
   ✓ build output validation > should not contain /src/scripts/ paths in built files  41748ms
 ✓ tests/scanSecretsScript.test.ts (2 tests) 83ms
 ✓ tests/helmChartReleaseVersion.test.ts (1 test) 4ms
 ✓ tests/changelogDocsAICoPilots.test.ts (1 test) 4ms
 ✓ tests/compactItemListHelpers.test.ts (2 tests) 5ms
 ✓ tests/customContentLoadTest.test.ts (2 tests) 1230ms
 Test Files  1 failed | 291 passed (292)
      Tests  1 failed | 1040 passed (1041)
   Start at  03:44:18
   Duration  160.08s (transform 9.56s, setup 10.98s, collect 45.85s, tests 116.63s, environment 189.18s, prepare 35.03s)


Error: Error: npm run format:check failed.

> dspace@3.0.0 format:check
> prettier --config .prettierrc --check "**/*.{js,ts,svelte,json,md}"

Checking formatting...
[warn] src/pages/quests/svelte/option/FinishOption.svelte
[warn] src/pages/quests/svelte/option/GotoOption.svelte
[warn] Code style issues found in 2 files. Run Prettier with --write to fix.

 ❯ scripts/tests/prettierFormatting.test.ts:17:13


 ELIFECYCLE  Command failed with exit code 1.
Error: Process completed with exit code 1.
```

new codex task, not a r/e/v/i/e/w task

--- a/frontend/src/pages/quests/svelte/option/FinishOption.svelte
+++ b/frontend/src/pages/quests/svelte/option/FinishOption.svelte
@@ -10,7 +10,11 @@
     export let quest, option;
     let githubConnected = false;
     const itemRequirementsMet = writable(
-        areItemRequirementsMet(option.requiresItems, get(gameStateStore)?.inventory, get(gameStateStore))
+        areItemRequirementsMet(
+            option.requiresItems,
+            get(gameStateStore)?.inventory,
+            get(gameStateStore)
+        )
     );
     let isDisabled = false;
 
@@ -23,13 +27,19 @@
     $: {
         if ($gameStateStore) {
             itemRequirementsMet.set(
-                areItemRequirementsMet(option.requiresItems, $gameStateStore.inventory, $gameStateStore)
+                areItemRequirementsMet(
+                    option.requiresItems,
+                    $gameStateStore.inventory,
+                    $gameStateStore
+                )
             );
         }
     }
 
     $: {
-        githubConnected = option.requiresGitHub ? isValidGitHubToken($gameStateStore?.github?.token) : false;
+        githubConnected = option.requiresGitHub
+            ? isValidGitHubToken($gameStateStore?.github?.token)
+            : false;
     }
 
     $: isDisabled = (option.requiresGitHub && !githubConnected) || !$itemRequirementsMet;

--- a/frontend/src/pages/quests/svelte/option/GotoOption.svelte
+++ b/frontend/src/pages/quests/svelte/option/GotoOption.svelte
@@ -9,7 +9,11 @@
     export let option, questId;
 
     const itemRequirementsMet = writable(
-        areItemRequirementsMet(option.requiresItems, get(gameStateStore)?.inventory, get(gameStateStore))
+        areItemRequirementsMet(
+            option.requiresItems,
+            get(gameStateStore)?.inventory,
+            get(gameStateStore)
+        )
     );
 
     function onClick() {
@@ -21,7 +25,11 @@
     $: {
         if ($gameStateStore) {
             itemRequirementsMet.set(
-                areItemRequirementsMet(option.requiresItems, $gameStateStore.inventory, $gameStateStore)
+                areItemRequirementsMet(
+                    option.requiresItems,
+                    $gameStateStore.inventory,
+                    $gameStateStore
+                )
             );
         }
     }

